packages/block-library/src/navigation/menu-items-to-blocks.js

```diff
@@ -189,9 +189,10 @@ function createDataTree( dataset, id = 'id', relation = 'parent' ) {
 			...data,
 			children: [],
 		};
-	}
-	for ( const data of dataset ) {
 		if ( data[ relation ] ) {
+			hashTable[ data[ relation ] ] = hashTable[ data[ relation ] ] || {};
```

Comment:
We also need to apply this fix to https://github.com/WordPress/gutenberg/blob/083e28d72b5449417086ebcad727e15d8d8e3f71/packages/edit-navigation/src/components/block-placeholder/menu-items-to-blocks.js#L97

Reply:
Good point! I also applied the fix in `packages/edit-navigation/src/store/menu-items-to-blocks.js`, from what I can tell it's the same thing in all 3 places :+1: